scenes/Tree1.tscn

```diff
@@ -3,7 +3,6 @@
 [ext_resource path="res://models/tree1.glb" type="PackedScene" id=1]
```

Comment:
So basically for this scene, you're bringing in the tree, defining some details about it and setting collision parameters - and that's it? Would you apply shaders in this step?

Reply:
I'm honestly not sure exactly what section adds shaders; this file is 100% generated by the IDE, and I have to kinda guess what anything does.

But yes, this file would, I believe, contain any shaders for objects in the scene. I think!